CardinalKit-Example/CardinalKit/Components/UIData.swift

```diff
@@ -168,6 +213,20 @@ class NotificationsAndResults: ObservableObject {
                 return Double(text.levenshtein(from: target)) / Double(target.count)
             case .amsler:
```

Comment:
Maybe leave a `#warning(TODO:)` so we remember to come back in the future?

Reply:
Sounds good